Contribute/adding-metadata.md

```diff
@@ -1,13 +1,13 @@
 # Annotating a document with meta data
 
-The documentation markdown files are allowed to contain meta data.  This is done by adding (YAML)[https://en.wikipedia.org/wiki/YAML] at the top of the document.
+The documentation markdown files are allowed to contain meta data.  This is done by adding [YAML](https://en.wikipedia.org/wiki/YAML) at the top of the document.
```

Comment:
Maybe we should not link to wikipedia, rather http://yaml.org/

Reply:
I think yaml.org is a bit geeky. Not understandable for non-tech People.